cmd/iam.go

```diff
@@ -1155,7 +1282,20 @@ func (sys *IAMSys) removeGroupFromMembershipsMap(group string) {
 
 // NewIAMSys - creates new config system object.
 func NewIAMSys() *IAMSys {
+	// Check global server configuration to determine the type of
+	// users system configured.
+
+	// The default users system
+	var utype UsersSysType
+	switch {
+	case globalServerConfig.LDAPServerConfig.ServerAddr != "":
```

Comment:
can you have some other boolean such that this code doesn't crash if `globalServerConfig` is nil ?

Reply:
Why do we need to handle iam init before server configuration is initialized? It seems like a reasonable requirement.

Reply:
There isn't I am simply talking about from tests point of view. @donatello